.github/workflows/build-test-virtual.yml

```diff
@@ -0,0 +1,34 @@
+name: "Build/test (virtual)"
+on: [push]
+jobs:
+  build-test:
+    runs-on: ubuntu-20.04
```

Comment:
nit: could this run on ubuntu 22.04 (just to use the latest LTS release) ?

Reply:
no we cannot use v22 until CCF fully tests and supports it